schemas/payload.json

```diff
@@ -195,9 +195,12 @@
       }
     },
     "env": {
-      "title": "Environment variable mappings.",
+      "title": "Environment variable mappings. Must be strings.",
       "description": "Example: ```\n{\n  \"PATH\": '/borked/path' \n  \"ENV_NAME\": \"VALUE\" \n}\n```",
-      "type": "object"
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
```

Comment:
It's non-trivial...  there is some logic here:
https://github.com/taskcluster/taskcluster-queue/blob/master/src/scan.js

But be very care not to accidentally delete everything!

IMO, it's safer and easier to just record all tasks using pulse exchanges... and then compare with a days worth of tasks... Or list a few task groups from treeherder...

`queue.listTaskGroup` can list everything in a group/push.